chapters/synchronous.tex

```diff
@@ -388,17 +388,29 @@ \section{Clock Constructors}\label{clock-constructors}
 The $\mathit{interval}$ must be strictly positive ($\mathit{interval} > 0$) of type \lstinline!Real! with \lstinline!unit = "s"!.
 The result is of base type \lstinline!Clock! that ticks when \lstinline!time! becomes $t_{\mathrm{start}}$, $t_{\mathrm{start}} + \mathit{interval}_{1}$, $t_{\mathit{start}} + \mathit{interval}_{1} + \mathit{interval}_{2}$, \@\ldots{}
 The clock starts at the start of the simulation $t_{\mathrm{start}}$ or when the controller is switched on.
-Here the next clock tick is scheduled at $\mathit{interval}_{1}$ = \lstinline!previous($\mathit{interval}$)! = \lstinline!$\mathit{interval}$.start!.
-At the second clock tick at time $t_{\mathrm{start}} + \mathit{interval}_{1}$, the next clock tick is scheduled at $\mathit{interval}_{2}$ = \lstinline!previous($\mathit{interval}$)!, and so on.
+Here the next clock tick is scheduled at $\mathit{interval}_{1}$ = \lstinline!$\mathit{interval}$!.
+At the second clock tick at time $t_{\mathrm{start}} + \mathit{interval}_{1}$, the next clock tick is scheduled at $\mathit{interval}_{2}$ = \lstinline!$\mathit{interval}$!, and so on.
 If $\mathit{interval}$ is a parameter expression, the clock defines a periodic clock.
 
 \begin{nonnormative}
-Note, the clock is defined with \lstinline!previous($\mathit{interval}$)!.
-Therefore, for sorting the input argument is treated as known.
 The given interval and time shift can be modified by using the \lstinline!subSample!, \lstinline!superSample!, \lstinline!shiftSample! and \lstinline!backSample! operators on the returned clock, see \cref{sub-clock-conversion-operators}.
 There are restrictions where this operator can be used, see \lstinline!Clock! expressions below.
 Note that $\mathit{interval}$ does not have to an evaluable expression, since different real interval clocks are never compared.
 \end{nonnormative}
+
+\begin{example}
+\begin{lstlisting}[language=modelica]
+  // first clock tick: previous(realNextInterval) = 0.002
+  Real realNextInterval(start = 0.002);
```

Comment:
Rather than having the very long name `realNextInterval` here, I think we could simply call it `nextInterval`, and at the same time update the example above to not use the name `nextInterval` for something which we call `intervalCounter` in the normative text.  Calling it `nextIntervalCounter` in that example would make more sense.